pkg/kn/commands/service/describe_test.go

```diff
@@ -432,28 +432,31 @@ func TestServiceDescribeVerbose(t *testing.T) {
 	r := client.Recorder()
 
 	// Prepare service
-	expectedService := createTestService("foo", []string{"rev1", "rev2"}, goodConditions())
+	expectedService := createTestService("foo", []string{"rev1", "rev2", "rev3"}, goodConditions())
```

Comment:
Do we have a test for the base case with one revision? I think there is but worth double checking. Thx

Reply:
We have the base case with one revision.